extensions/Vendor/EXT_texture_transform/schema/EXT_texture_transform.textureInfo.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "EXT_texture_transform textureInfo extension",
+    "type" : "object",
```

Comment:
Is it intentional that this extension object doesn't inherit from `gltfProperty`, i.e., lacks `extras` and `extensions`?

Reply:
Oh, I didn't realize that the glTF semantics supported extending extensions. Doesn't that make the difficult job of loading extended properties even harder? I can add that if desired.

Reply:
The SpecGloss extension (the only ratified 2.0 extension) follows that.

Reply:
It's added as of 60dcf94

Reply:
For schema to be correct, `extensions` and `extras` need to be mentioned in `properties` (see other schema files for example).

We don't have any guideline on whether or not allow extending extensions. I just think that it's a bit better to have uniform approach to all glTF objects.

Reply:
Added as of 0b22fc5